index.html

```diff
@@ -571,70 +571,98 @@ <h3>External Binary Content</h3>
         <blockquote id="message-external-body-variability" class="informative">
```

Comment:
We should probably change the `id` to something like: `external-content-variability`.

Reply:
Yup, I didn't see that... will update